application/helpers/admin/token_helper.php

```diff
@@ -14,171 +14,72 @@
 
 /**
 * Sends email to tokens - invitations, reminders, registers, and confirmations
-*
+* Seems used only in remote_control : maybe move it to ?
 * @param integer $iSurveyID
 * @param CActiveRecord[]  $aResultTokens
 * @param string $sType type of notification invite|register|remind|confirm
 * @return array of results
 */
 function emailTokens($iSurveyID, $aResultTokens, $sType)
 {
-    Yii::app()->loadHelper('common');
```

Comment:
Hi ho! This function is so different in dev compared to master it's not possible to merge Dominik's ONE-LINE change without major conflicts. Can you have a look?

Reply:
This is the fix:

`$fieldsarray["{SID}"] = $iSurveyID`

but fieldsarray is removed.